configure.ac

```diff
@@ -914,16 +917,13 @@ A8_OPTION(pbi_bb,yes,
      )
 AM_CONDITIONAL([WANT_PBI_BB], test "$WANT_PBI_BB" = "yes")
 
+if [[ "$SUPPORTS_NETSIO" = "yes" ]]; then
 dnl NetSIO / FujiNet support (only on Linux or MacOS)
-    A8_OPTION(netsio,no,
-            [Support NetSIO Protocol I/O interface (FujiNet) (default=OFF)],
-            NETSIO,[Define to enable NetSIO Protocol/FujiNet support.]
-            )
+A8_OPTION(netsio,no,
```

Comment:
Indentation (at some places `dnl` is aligned with `if` but AFAICS `A8_OPTION` is always indented).